polygon-digital-carbon/subgraph.template.yaml

```diff
@@ -3,11 +3,6 @@ description: Polygon Carbon
 repository: https://github.com/KlimaDAO/klima-subgraph
 schema:
   file: ./schema.graphql
-features:
-  - grafting
-graft:
-  base: QmUTK4dB7CXVvN2XsX1temzZGcryQ59j8YKdQP5Z2AwzVZ
-  block: 58282296
```

Comment:
This needs to be rebased, as of the other PR we now have a commented out `graft` section

Reply:
The rebase was up to date but I reset to the commit I removed the graft and commented it out there instead vs adding more commits to add + comment out